nixos/modules/services/monitoring/prometheus/exporters/folder-size.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, options, utils }:
+let
+  cfg = config.services.prometheus.exporters.folder-size;
+
+  settingsFormat = pkgs.formats.json { };
+  settingsFile = settingsFormat.generate "config.json" (builtins.filter (folder: builtins.hasAttr "path" folder) cfg.folders);
+in
+{
+  port = 9974;
+  extraOpts = {
+    folders = lib.mkOption {
+      description = "Folders to Scan";
```

Comment:
An example for `folders` option would be great

Reply:
An example as part of teh option I presume?

Reply:
I meant `example = [ { ... } ];`